File: exp1_ProgrammingBasics/nov_1_3.m
function res = nov_1_3(threshold)

    res = [];

    for a = 1:threshold
        if mod((a ^ 2 - a), 100) == 0
            b = (a ^ 2 - a) / 100;
            if b >= 1 && b <= threshold
                res = [res; a, b];
            end
        end
    end

    return
end

File: exp1/nov_1_3.py
def get_solution(threshold: int) -> list:
    res = []

    for a in range(1, threshold + 1):
        if not (a ** 2 - a) % 100:  # b is an integer
            if 1 <= (a ** 2 - a) // 100 <= threshold:
                res.append([a, (a ** 2 - a) // 100])

    return res